client-app/src/examples/contact/mobile/DirectoryPanelModel.ts

```diff
@@ -0,0 +1,120 @@
+import {HoistModel, managed, persist, XH} from '@xh/hoist/core';
+import {bindable, makeObservable, observable, action, runInAction} from '@xh/hoist/mobx';
+import {GridModel} from '@xh/hoist/cmp/grid';
+import {nameCol, locationCol} from '../../../core/columns';
+import {uniq} from 'lodash';
+
+// Shared from the desktop version
+import {PERSIST_APP} from '../AppModel';
+
+import ContactDetailsModel from './details/DetailsPanelModel';
```

Comment:
Importing this under a different name to the class is confusing - let's use the same name in both the definition and its usages.

Reply:
This must have been a missed update. Originally the files had bespoke names in the mobile app, but in a second pass I unified them with the desktop naming. Must have missed this change to the name reference. Ill update.